modules/exploits/linux/http/artica_proxy_rce.rb

```diff
@@ -0,0 +1,136 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Artica proxy 4.30.000000 Remote Code Execution',
+        'Description' => %q{
+          This module exploits an authenticated command injection vulnerability in Artica Proxy, combined with an authentication bypass
+          discovered on the same version, it is possible to trigger the vulnerability without knowing the credentials.
+          The webserver usually runs as root, successful exploitation of this vulnerability yields remote code execution as root on the
+          remote system.
```

Comment:
Perhaps I'm wrong, but as far as I'm aware this software is only distributed as a virtual appliance (?). In which case it is safe to assert that the web server runs as root.